pkg/storaged/stratis-details.jsx

```diff
@@ -53,6 +54,14 @@ import { std_reply, with_keydesc, with_stored_passphrase } from "./stratis-utils
 
 const _ = cockpit.gettext;
 
+export function check_stratis_warnings(client, enter_warning) {
+    for (const p in client.stratis_pools) {
+        const blockdevs = client.stratis_pool_blockdevs[p] || [];
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230707-120835-5b4a3bfc-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)